jlh/WeatherApp/app/src/main/res/layout/activity_detail.xml

```diff
@@ -9,6 +9,8 @@
     android:paddingRight="@dimen/weather_item_horizontal_spacing"
```

Comment:
Why do we have these padding values here? 
![screen shot 2017-04-03 at 11 10 46](https://cloud.githubusercontent.com/assets/4919888/24602688/355a1cc2-185e-11e7-9d49-6b5662774e0a.png)